### ISSUE TITLE
Crash with invalid id loads after updating to the new version.

### ISSUE THREAD

**Issue by panskdev** [copycats-plus/copycats#162]
The modloader seems to load the mods but eventually crashes and outputs this msg:
```The game crashed whilst initializing game
Error: java.lang.RuntimeException: One of more entry values did not copy to the correct id. Check log for details!
```

**To Reproduce**
1. Update copycats to the recent version
2. click on "launch"
3. See error

**Expected behavior**
Normal mod-loading procedure.

**Screenshots**
![image](https://github.com/user-attachments/assets/1061d4b9-d05c-47c5-ab7e-3d548419a425)

**Desktop (please complete the following information):**
 - Minecraft Version: **1.20.1**
 - Mod Loader : **Forge**
 - Mod Version **4.7.3** (tried with **4.7.10** (latest) too)

crash link :
https://mclo.gs/9dE623L

**Comment by Bennyboy1695** [comment]
Can we possibly get your latest.log for this crash please? If you've already downgraded the version again it will be one of the .gz files. So it might be easier to upgrade again to get the log

**Comment by panskdev** [comment]
sure here's the link : https://mclo.gs/qWJNo68

**Comment by hlysine** [comment]
Thanks for the report. This has been fixed in v2.1.4
